Model/Ui/ConfigProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Airwallex\Payments\Model\Ui;
+
+use \Magento\Checkout\Model\ConfigProviderInterface;
+use \Magento\Framework\App\Config\ScopeConfigInterface;
+use \Magento\Payment\Model\MethodInterface;
+
+class ConfigProvider implements ConfigProviderInterface
+{
+    const MODE_PATH = 'payment/airwallex_payments_basic/mode';
+
+    const CAPTURE_PATH = 'payment/airwallex_payments_card/airwallex_payment_action';
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * ConfigProvider constructor.
+     * @param ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        ScopeConfigInterface $scopeConfig
+    ) {
+        $this->scopeConfig = $scopeConfig;
+    }
+
+    /**
+     * Adds mode to checkout config array
+     *
+     * @return \array[][]
+     */
+    public function getConfig()
+    {
+        $config = [
+            'payment' => [
+                'airwallex_payments' => [
+                    'mode' => $this->scopeConfig->getValue(self::MODE_PATH) ?? 'demo',
```

Comment:
Why a default value in this getValue method? Cant the default value be set in the config (and enforced?). This can create unwanted side-effects. (especially if we rename demo for example)

Reply:
After fresh module installation if the mode is not yet set null value is retrieved. Will proceed to add a default value in config.xml. It will solve this issue.